src/XrdSciTokens/XrdSciTokensRedir.hh

```diff
@@ -0,0 +1,43 @@
+#pragma once
```

Comment:
@abh3 @bbockelm There are only 4 headers currently using this. For consistency, I think we should convert them to normal header guards (as well as this one). What do you think?

Reply:
I would agree.